### ISSUE TITLE
wow just wow

### ISSUE THREAD

**Issue by Wladefant** [SeptemberHX/joplin-plugin-enhancement#52]
@personalizedrefridgerator if you could try to integrate some of the features in this repo directly into joplin, the app would become so much nearer to obsidian and in some regards even better.

**Comment by Wladefant** [comment]
Laurent talked about integrating it too. https://discourse.joplinapp.org/t/plugin-many-enhancement-for-my-daily-usage-v1-2-0-2023-03-30/25622/7

**Comment by SeptemberHX** [comment]
I'm glad you like the plugin!

However, it's challenging for me to directly submit certain features to Joplin or have them included as default plugins. This is because Joplin has its own coding style, file structure, and more stringent quality requirements.

To be able to merge them into Joplin, on one hand, the plugin needs to be widely used. On the other hand, further improvement in code quality is necessary. Currently, both of these aspects are quite difficult to achieve.

**Comment by Wladefant** [comment]
Yes for sure. And some of the features are also no good for the default consumer. But I hope @personalizedrefridgerator can you help you there. In the last months he was the one creating cool features like jsdraw and seems to be quite active.

**Comment by personalizedrefrigerator** [comment]
We're currently working on [including Simple Backup](https://github.com/laurent22/joplin/pull/9360) as a default plugin.

Using the same mechanism, `joplin-plugin-enhancement` could be bundled with the desktop app without being a part of the main Joplin codebase (it would still be maintained here).

There will probably be a post on the Joplin Forum to discuss which plugins should be included by default in the next few weeks/months.

**Comment by Wladefant** [comment]
Oh that is even better, because when the code in joplin grows but the staff not that is no good. The whole enhance plugin should not be added as default but definitely some parts of it

**Comment by Wladefant** [comment]
I now wrote a long blog post of my goals and hopes https://discourse.joplinapp.org/t/an-innovative-approach-to-joplins-viewers-and-editors-tailored-for-every-user/33859?u=wladefant
